examples/events/EiffelIssueDefinedEvent/simple.json

```diff
@@ -0,0 +1,24 @@
+{
+	"meta": {
+		"type": "EiffelIssueDefinedEvent",
+		"version": "1.0.0",
+		"time": 1234567890,
+		"id": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0"
+	},
+	"data": {
+		"type": "BUG",
+		"tracker": "DONE", //the hell's a tracker?
+		"id": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0",
+		"uri": "http://zombo.com/"
```

Comment:
`"uri": "https://github.com/my-user/my-repo/issues/3"`

Reply:
But at zombo com you can do anything.

Reply:
I'm afraid I don't understand the reference, but we should use something illustrative.